test/update_dev_build.sql

```diff
@@ -0,0 +1,84 @@
+-- adding  accounts,  for suppliers
+
+INSERT INTO account (type_id, enterprise_id, number, parent, label)
+VALUES (2, 1, 40111002 ,0, "SUPPLIER'S ACCOUNT 1"),
+        (2, 1, 40111003 ,0, "SUPPLIER'S ACCOUNT 2");
+
+
+-- creditor group
+INSERT INTO creditor_group (enterprise_id, uuid,name, account_id, locked)
+VALUES (1,'cgsp1', 'Creditor groud supplier1', 286,0),
+        (1, 'cgsp2', 'Creditor groud supplier2', 287,0);
```

Comment:
- The `uuid` column is a BINARY(16) : 

There are some useful functions for handling uuid with BINARY(16) type,

- UUID() : Generate a uuid string like `e2da4b1a-19c7-4903-8399-89ec481ead2c`

- HUID('e2da4b1a-19c7-4903-8399-89ec481ead2c') : Convert the uuid string 'e2da4b1a-19c7-4903-8399-89ec481ead2c' into a binary `0x00EFB27B0D504561BF1394337C069C2`

- BUID(0x00EFB27B0D504561BF1394337C069C2) : Convert a binary uuid to a string uuid `e2da4b1a-19c7-4903-8399-89ec481ead2c`, it is the opposite of HUID()

For the query above, you can write : 
```sql
INSERT INTO creditor_group (enterprise_id, uuid,name, account_id, locked) VALUES 
(1, HUID(UUID()), 'Creditor groud supplier1', 286,0),
(1, HUID(UUID()), 'Creditor groud supplier2', 287,0);
```

If the uuid can be used in other places, you can create a kind of variable : 
```sql
SET @CreditorGroup1 = HUID(UUID());
SET @CreditorGroup2 = HUID(UUID());

INSERT INTO creditor_group (enterprise_id, uuid,name, account_id, locked) VALUES 
(1, @CreditorGroup1, 'Creditor groud supplier1', 286,0),
(1, @CreditorGroup2, 'Creditor groud supplier2', 287,0);
```

- The `account_id` must be set above.